part_test.go

```diff
@@ -328,7 +328,7 @@ func TestMultiMixedParts(t *testing.T) {
 		Parent:      test.PartExists,
 		NextSibling: test.PartExists,
 		ContentType: "text/plain",
-		Charset:     "us-ascii",
+		Charset:     "ISO-8859-1",
```

Comment:
I'm not so sure about this. Looking at the part in question (`multimixed.raw`), it appears that it is indeed `us-ascii`. Sure, `ISO-8859-1` (should that be lowercase?) will work, but so would `utf-8`.

Thoughts, @jhillyerd?

Reply:
I think they are the same when there is no 8-bit characters present.  Based on [this table](http://www.columbia.edu/kermit/ascii.html) it seems the only 8-bit characters in us-ascii are control codes that wouldn't be present in an email.

I think that means there would be zero impact to how a `us-ascii` detected as `iso-8859-1` message is decoded to UTF-8, which is our goal.

If we do start seeing problems, we can look at work-arounds or forking the [chardet package](https://github.com/gogs/chardet) (it's ancient).